.github/workflows/e2e-arm64.yaml

```diff
@@ -0,0 +1,41 @@
+name: E2E-arm64
+on: [push, pull_request]
+permissions: read-all
+jobs:
+  goversion:
+    uses: ./.github/workflows/go-version.yaml
```

Comment:
Go version no longer comes from a re-usable workflow. Please update workflows to reflect the changes from https://github.com/etcd-io/etcd/pull/16088